src/main/java/org/projectempire/lx/osc/QLabClips.java

```diff
@@ -0,0 +1,97 @@
+package org.projectempire.lx.osc;
+
+import heronarts.lx.LXPlugin;
+import heronarts.lx.mixer.LXAbstractChannel;
+import heronarts.lx.osc.LXOscEngine;
+import heronarts.lx.osc.LXOscListener;
+import heronarts.lx.parameter.LXParameterListener;
+import heronarts.lx.LX;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+
+@LXPlugin.Name("QLab OSC Listener")
+public class QLabClips implements LXPlugin, LXOscListener, LXParameterListener {
+
+    public static final int RECV_OSC_PORT = 53001;
+    public static final int QLAB_OSC_PORT = 53000;
+    public static final String OSC_IP = "192.168.4.41";
```

Comment:
I don't think this IP is anything on our network. We are 192.168.3.x, with the qlabs machine at .11 I think